content/extraProblems.json

```diff
@@ -705,6 +705,71 @@
         "usacoId": "1190"
       }
     },
+    {
+      "uniqueId": "usaco-1204",
+      "name": "Photoshoot 2",
+      "url": "http://usaco.org/index.php?page=viewproblem2&cpid=1204",
+      "source": "Bronze",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Brute Force"],
```

Comment:
^ why?

Reply:
idk what should i put it as? would ad hoc be better?

Reply:
just don't tag it at all? that's what we've been doing so far for ad hoc problems